scrapers/Baberotica.yml

```
@@ -0,0 +1,36 @@
+name: Baberotica
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - avidolz.com
+      - baberotica.com
+      - baberoticavr.com
+      - teenthais.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title:
+        selector: //h1[@itemprop="name"]
+      Performers:
+        Name: //p/span/a/span
```

Although it works i would suggest something like
```suggestion
        Name: //p/span[@itemprop="actor"]/a/span
```
as a little more specific/safe